cmd/ocm/login/cmd.go

```diff
@@ -295,11 +278,39 @@ func run(cmd *cobra.Command, argv []string) error {
 		clientID = args.clientID
 	}
 
-	// If the value of the `--url` is any of the aliases then replace it with the corresponding
-	// real URL:
-	gatewayURL, ok := urlAliases[args.url]
-	if !ok {
+	// URL Precedent (from highest priority to lowest priority):
+	//   1. runtime `--url` cli arg (key found in `urlAliases`)
+	//   2. runtime `--url` cli arg (non-empty string)
+	//   3. config file `URL` value (non-empty string)
+	//   4. sdk.DefaultURL
+	gatewayURL := sdk.DefaultURL
```

Comment:
Do you think we should add this to a function? We are re-using this in list rh-region command as well